incubator/hnc/test/e2e/issues_test.go

```diff
@@ -161,18 +161,6 @@ var _ = Describe("Issues", func() {
 		RunShouldContain("ActivitiesHalted (ParentMissing)", defTimeout, "kubectl hns describe", nsChild)
 	})
 
-	It("Should delete namespace with ParentMissing condition - issue #716", func() {
```

Comment:
Why did you remove this?

Reply:
I looked at issue #716 and feel like it's not relevant anymore?

Reply:
sgtm